### DISCUSSION TITLE
安卓版火狐浏览器“代理服务器拒绝连接”

### DISCUSSION THREAD

**Discussion by voiotikos** [immersive-translate/immersive-translate#2361] [Help]
### 概述 | Summary

因为安卓上的插件只有火狐浏览器有，但火狐浏览器一直出现“代理服务器拒绝连接”，根本无法访问网页，也就没办法用沉浸式翻译的插件。是否有其它方法？

**Comment by theowenyoung** [comment]
你还可以用 安卓上的 Edge 浏览器。https://immersivetranslate.com/zh-Hans/docs/installation/#%E9%80%9A%E8%BF%87-edge%E5%BE%AE%E8%BD%AF-%E5%AE%89%E5%8D%93%E6%B5%8F%E8%A7%88%E5%99%A8%E5%AE%89%E8%A3%85
